cdc/cdc_options.hh

```diff
@@ -61,7 +62,7 @@ public:
     std::map<sstring, sstring> to_map() const;
     sstring to_sstring() const;
 
-    bool enabled() const { return _enabled; }
+    std::optional<bool> enabled_opt() const { return _enabled; }
```

Comment:
I think it might be better to keep `bool enabled()` and just add `bool is_enabled_set()`. That would reduce the change significantly and also the interface would be nicer not requiring all the places do call `value_or`.